### PR TITLE
Update FTO template to include a guide on GitHub

### LINKED ISSUE #7069: Update FTO template to include a guide/tutorial on GitHub and Pull Requests
A lot of beginners who are willing to contribute are not familiar with GitHub. I think we should update the FTO template to include above mentioned guides to encourage contributions from such people.

### PR DESCRIPTION
**Pull request by hedonhermdev**
A lot of beginners are not familiar with how GitHub and Pull Requests work. To encourage contributions from such people, I added a link to a guide from GitHub itself which explains the process of making a pull request.

Fixes #7069 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by hedonhermdev** [comment]
If anyone knows of a better guide feel free to mention it here and I will update the PR.

File: .github/first-timers-issue-template.md [hunk 1]
```diff
@@ -28,10 +28,10 @@ $DIFF
 - [ ] 💾 **Commit** your changes
 
 - [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:
+1. If you are not familiar with GitHub or the pull request model, [here is a guide you can follow](https://guides.github.com/activities/hello-world/) on how GitHub works.
+2. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.
```
Comment (pythongiant) [inline .github/first-timers-issue-template.md]: Could you change it to 
``` If you are familiar with the terminal or would like to learn to use it ```

Comment (hedonhermdev) [reply .github/first-timers-issue-template.md]: I agree. Will change it.

**Comment by jywarren** [comment]
Thank you so much!!! And thanks @ananya for your help supporting this!

**Pull request merged**